### ISSUE TITLE
HIndi Rendering Problem

### ISSUE THREAD

**Issue by MayankFawkes** [signintech/gopdf#293]
Incorrent rendering Hindi fonts https://hindi-fonts.com/fonts/mangal-regular

```go
pdf := gopdf.GoPdf{}
pdf.Start(gopdf.Config{PageSize: *gopdf.PageSizeA4})
pdf.AddPage()
err := pdf.AddTTFFont("mangal", "mangal.ttf")
if err != nil {
    log.Print(err.Error())
    return
}

err = pdf.SetFont("mangal", "", 14)
if err != nil {
    log.Print(err.Error())
    return
}
pdf.Cell(nil, "नमस्ते")
pdf.WritePdf("hello.pdf")
```

Output

![image](https://github.com/golang/go/assets/43340190/4ecf3fc3-5fb2-4632-aab8-047533ae471d)

Expected 

![image](https://github.com/golang/go/assets/43340190/984122cf-9ea8-41d6-a070-a182b2d1ef6f)

**Comment by oneplus1000** [comment]
I'm not 100% sure, but I think this problem is caused by combine 2 letters in to one letters. This problem is more specific to the language than the gopdf lib that tries to act as a simple library for generating PDF will do. 

And I've seen a similar solution in Arabic: they take the string through a conversion function before sending it to gopdf.
```GO
pdf.Text(reverseString(goarabic.ToGlyph("الايام السبعة")))
```

**Comment by oneplus1000** [comment]
I don't mean that [goarabic](https://github.com/01walid/goarabic) can be used in Hindi. What I mean is that the problem arises the same way as in Arabic, where letters are changed or combined depending on the preceding letter. What you should do is write a character converter similar to goarabic but in hindi.

**Comment by MayankFawkes** [comment]
yeah i am looking for something like that i found [go-text/typesetting](https://github.com/go-text/typesetting) but not sure if we can use it or not or how can we use it so i made issue there https://github.com/go-text/typesetting/issues/159 lets see
